src/data/guidelines/color-tokens.js

```diff
@@ -881,8 +881,8 @@ const colorTokens = {
           hex: '#e0e0e0',
         },
         g10: {
-          name: 'Gray 30',
```

Comment:
Do we want to be changing this / removing the `border-tile` tokens?

Reply:
😓 None of these changes are supposed to be in this PR. Only the usage.mdx, style.mdx, and images for `Slider` have been changed in this PR. I am not sure why this keeps happening.

Reply:
Okay I'll remove those changes

Reply:
Should be good now, let me know if everything looks correct

Reply:
Yes! This all looks correct now. Thank you!